lib/src/page_list_viewport_gestures.dart

```diff
@@ -288,35 +305,190 @@ class _PageListViewportGesturesState extends State<PageListViewportGestures> wit
   }
 }
 
-class DeprecatedPanAndScaleVelocityTracker {
-  /// The maximum velocity a gesture can have and still be considered a viewport
-  /// re-adjustment. Used in tandem with the [kViewportReAdjustmentMinTranslationDistance]
-  /// to determine if the user is re-adjusting the viewport. Viewport re-adjustments
-  /// result in the momentum simulation to be aborted.
-  static double kViewportReAdjustmentMaxVelocity = 600;
+/// Definiton for gestures' translation distance and velocity categories.
+///
+/// Distances are tiny, small, and large.
+/// Velocities are slow, normal, and fast.
+/// These categories are used to individually define ballistic simulation behavior
+/// across a variety of scrolling situations.
+class GestureThresholdsAndScales {
+  /// Definition for a tiny distance in pixels.
+  ///
+  /// {@template distance_definitions}
+  /// Gesture translation distance categorization.
+  /// The launch velocity for the ballistic simulation can be individually
+  /// scaled for gestures categorized into these categories.
+  /// Distances scale diagram:
+  /// (0 ... "tiny" ... tinyDistanceMax] (... "small" ... SmallDistanceMax]( ... "large" ...
+  /// {@endtemplate}
+  static const double tinyDistanceMax = 3;
+
+  /// Definition for a small distance in pixels.
+  ///
+  /// {@macro distance_definitions}
+  static const double smallDistanceMax = 120.0;
+
+  /// Definition for a slow velocity in pixels per second.
+  ///
+  /// {@template velocity_definitions}
+  /// Gesture velocity categorization.
+  /// The launch velocity for the ballistic simulation can be individually
+  /// scaled for gestures categorized into these categories.
+  /// Velocities scale categorization diagram:
+  /// (0 ... "slow" ... slowVelocityMax] (... "normal" ... normalVelocityMax]( ... "fast" ...
+  /// {@endtemplate}
+  static const double slowVelocityMax = 300.0;
+
+  /// Definition for a normal velocity in pixels per second.
+  ///
+  /// {@macro velocity_definitions}
+  static const double normalVelocityMax = 850.0;
+
+  /// Minimal neccessary velocity when the user releases from any panning motion
+  /// required for which a ballistic simulation to be launched.
+  ///
+  /// Value is in pixels per second.
+  static const double minSmallTranslationBallisticActivationVelocity = 120.0;
+
+  /// {@template velocity_increase}
```

Comment:
The entirety of a Dart Doc should never be a template because it means you're using the same description for multiple properties or methods or classes, which should never be the case.